paddle/fluid/framework/grad_op_desc_maker.h

```diff
@@ -186,6 +186,42 @@ class DefaultGradOpDescMaker : public SingleGradOpDescMaker {
   }
 };
 
+// CheckInputGradOpDescMaker allowed user to override the CheckInput virtual
+// functions. user can customize the input grad he need to save memory.
+template <bool DropEmptyIG = true>
+class CheckInputGradOpDescMaker : public SingleGradOpDescMaker {
```

Comment:
please read our op develop standard, and I think this class is useless.

Reply:
removed